README.md

```
@@ -32,25 +31,26 @@ Generate types
 pn types
 ```
 
-Deploy to cloudflare
+sync astro
 ```shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)